### DISCUSSION TITLE
[Question]

### DISCUSSION THREAD

**Discussion by Nubinator** [kubilus1/autoortho#260] [Q&A] [answered]
### Please check existing docs and issues first.

- [X] I have checked for existing answers
- [X] This is not a bug or other "Issue"
- [X] I have read the FAQ and documentation

### System Details

- OS: Windows 11
- XPlane Version: XP12
- AutoOrtho Version: 
- Filesystem type: NTFS

### question

Is there a way to not use auto ortho temporarily, there are few airports that i have my own mesh files for but with auto ortho enabled they seem to glitch, is there a way to just stopping autoortho working for a while apart from moving the actual files out of the folder or putting "_DISABLED" to every single file and then removing it?

**Comment by dpdurst** [comment]
Nope

**Reply by rayw45** [reply, marked as answer]
From what I understand if your related Mesh files are placed further down  your scenery+packs.ini file list after sim heaven but just before autoortho lines that to me seems to work okay certainly for the Brisbane City and airport scenery files. 

Don't take my word for it as I am no expert but just stating what I have found with experimenting with the Scenery_packs.ini file listing order.

So far though with the Bribane City scenery I still see no traffic on the roads in the scenery covered area which makes it look a bit like a ghost town, beautiful but dead.  Currently looking for a solution to that with the developers via Orbyx.

**Reply by dpdurst** [reply]
" is there a way to just stopping autoortho working for a while apart from moving the actual files out of the folder or putting "_DISABLED" to every single file and then removing it"

You asked the above, not about using other Orthos with AO thus this was my answer. If you had said you wanted to use other Orthos at the same time I may have given a better answer. Much information about this on the Xplane forums.

**Comment by Nubinator** [comment]
> From what I understand if your related Mesh files are placed further down your scenery+packs.ini file list after sim heaven but just before autoortho lines that to me seems to work okay certainly for the Brisbane City and airport scenery files.
> 
> Don't take my word for it as I am no expert but just stating what I have found with experimenting with the Scenery_packs.ini file listing order.
> 
> So far though with the Bribane City scenery I still see no traffic on the roads in the scenery covered area which makes it look a bit like a ghost town, beautiful but dead. Currently looking for a solution to that with the developers via Orbyx.

Thank you, that seemed to fix it, its not perfect as there are some large holes glitches holes near the airport but i dont really mind it. Thanks again

**Reply by rayw45** [reply]
Glad it helped.  Those large  holes I do not see but I may  just not have noticed them.  Usually this is due to autoortho not yet been able to download a minimum ZL tile for that spot and uses the default green tile I understand.   Make sure as recommended on this site that the AO Min ZL=12.  The important word here is "minimum" as I believe that AO will still load higher resolution ZL tiles if available within I think the max wait time that is set in the config file.  Mine is set to 2 seconds and now unless I am flying over about 300kts GSP I do not see many and sometimes no pauses during flights over areas I have not flown before or at least since installing AO.  

If I am not mistaken if you are flying fast say in an airliner over areas you have flown before and there are already downloaded minimum ZL tiles in your cache folder then I do not think you should see any pauses.  Sure it will happen over areas without any previously downloaded tiles in your cache unless you are flying really slow.  That is my thoughts but maybe one of the experts here can confirm or correct me.

Yes those mesh file positions seem to be important. I do not know how they work but assume they mesh different scenery areas together smoothly but I may be wrong.  Anybody can explain how the mesh files work please ??
